### ISSUE TITLE
Participation in FMU Cross Check

### ISSUE THREAD

**Issue by chrbertsch** [modelon-community/fmi-library#5]
It would be highly beneficial if FMI-Library would participate in the FMI Cross Check https://github.com/modelica/fmi-cross-check. For many tools this has significantly improved the maturity of their FMI support.

**Comment by filip-stenstrom** [comment]
FMIL doesn't provide any master algorithm for simulating arbitrary FMUs, so it's not an "importing tool" (it's just a library). However, if any importing tool that's using FMIL is passing XC, then the used version of FMIL should also "pass".

We could perhaps make references in the project README to importing tools that use FMIL, and that are listed in XC. This would not show up in the XC tables though. Any thoughts @iakovn ?

**Comment by chrbertsch** [comment]
Hello Filip, you have examples like https://github.com/modelon-community/fmi-library/blob/master/Test/FMI2/fmi2_import_me_test.c where you simulate FMUs. Couldn't you adapt this for the XC FMUS and report the result??
Typically, in the FMI cross-Check , the more fundamental questions like errors in the XML are detected and not deep numerical problems that would require an advanced solver (for ME FMUs) or master (for CS FMUs; in the XC we deal with single FMUs anyway). The information, which FMUs can be treated with FMIL would be very valuable!

**Comment by iakovn** [comment]
I agree with Filip that FMIL that unless we include at least a basic simulation + master algorithm + input/output handling into the library reporting XC results is a bit misleading. I think that we should rather indicate that pyfmi is relying on FMIL when reporting the results to XC in some way.
@chria - what do you think?

**Comment by chria** [comment]
@iakovn I think this is reasonable. I'm in favor of indicating that PyFMI relies on FMIL and that the XC results of PyFMI points also to the maturity of FMIL.
